bot/pagination.py

```diff
@@ -191,7 +191,7 @@ def _split_remaining_words(self, line: str, max_chars: int) -> t.Tuple[str, t.Op
     async def paginate(
         cls,
         lines: t.List[str],
-        ctx: Context,
+        ctx: t.Union[Context, discord.Interaction],
```

Comment:
```suggestion
        ctx: Context | discord.Interaction,
```

this can be done for the rest by ditching the typing import as well, if you want :p
Example: `list[str]` will work too